File: app/src/main/java/br/ufpa/smartufpa/models/overpass/OverpassModel.java

package br.ufpa.smartufpa.models.overpass;

import java.util.List;
import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class OverpassModel {

    @SerializedName("elements")
    @Expose
    private List<Element> elements;

    public List<Element> getElements() {
        return elements;
    }

    public boolean isEmpty(){ return getElements().isEmpty();}



}

File: app/src/main/java/br/ufpa/smartufpa/models/overpass/OverpassModel.kt
package br.ufpa.smartufpa.models.overpass

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class OverpassModel {

    @SerializedName("elements")
    val elements: List<Element>? = null

    val isEmpty: Boolean
        get() = elements!!.isEmpty()


}
